nextcloud_install_production.sh

```diff
@@ -704,9 +704,14 @@ rm -f *.zip.*
 done;
 
 # Install virtual kernels
-apt install linux-tools-virtual-hwe-16.04-edge linux-cloud-tools-virtual-hwe-16.04-edge  -y
-apt install linux-image-virtual-hwe-16.04-edge -y
-apt install linux-virtual-hwe-16.04-edge -y
+apt purge -y $(dpkg-query -W -f='${binary:Package}\n' 'linux-image-4*' 'linux-headers-4*' | grep -v 4.4.0-64-generic)
```

Comment:
This is part is kind of werid.
Ubuntu already has a 4.4.0-65 but if you install

`linux-image-virtual-lts-xenial linux-headers-virtual-lts-xenial`

 It will install the 4.4.0-64 version, **_which happens to be the one with aufs support_**, the -65 OTOH, doesn't.
I think that i'll add 
`grep aufs /proc/filesystems`

if it returns aufs, then i shouldn't touch the kernels, but if the return is empty it should try to configure the aufs support, by modifying the kernels and stuff, or something like it.